sgkit/stats/regenie.py

```diff
@@ -19,6 +19,14 @@
 )
 
 
+def _map_blocks_asnumpy(x: Array) -> Array:
+    if da.utils.is_cupy_type(x._meta):
```

Comment:
```suggestion
    if da.utils.is_cupy_type(x._meta):  # pragma: no cover
```

Reply:
Thanks @tomwhite for the suggestion, I did miss that failure but I applied the change now.